lib/src/main/java/com/futurewei/alcor/common/db/ignite/MockIgniteServer.java

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.common.db.ignite;
+
+import org.apache.ignite.Ignite;
+import org.apache.ignite.IgniteException;
+import org.apache.ignite.Ignition;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+public class MockIgniteServer {
+    private static Ignite igniteServer;
+
+    @BeforeClass
+    public static void init() {
+        try {
+            igniteServer = Ignition.start();
```

Comment:
Could we change the default Ignite port to some numbers other than 10800 and use that new default number in the application.properties? This could prevent conflicting with a running Ignite container on the 10800 port.

Reply:
That's a good idea. Let me check the ignite doc.

Reply:
Thanks for making the change.